src/Symfony/Component/Lock/Store/SemaphoreStore.php

```diff
@@ -98,10 +101,18 @@ public function delete(Key $key)
     }
 
     /**
-     * {@inheritdoc}
+     * Extends the ttl of a resource.
```

Comment:
Is this docblock needed? What's about other stores?

Reply:
I need to add it on other stores